js/game.js

```diff
@@ -0,0 +1,307 @@
+                if(previous_card === c1 ||
+                    previous_card === c2) {
+                    pair.found = true;
+                    changeScore(true);
+                } else {
+                    cards_flipped -= 2;
+                    wrong_pair = true;
+                    changeScore(false);
+                }
+
+                if(cards_flipped === 16) {
+                    setCardBrightness(100);
+
+                    if(finishGame()){
+                        start();
+                    }
+                }
+                
+                return;
+            }
+        }
+    }
+}
+
+/*
+    Creates and inserts a Pair object to its
+    respective array.
+*/
+function createPair(card_one_id, card_two_id){
+    existing_pairs.push({
+        card_one : img_array[card_one_id],
+        card_two : img_array[card_two_id],
+        src : setRandomImage(),
+        found : false
+    });
+}
+
+function move(event) {
+    if (event.defaultPrevented)
+      return; // Do nothing if event already handled
+
+    setCardBrightness(100);
+
+    let direction = event.code;
+    let id = parseInt(selected_card.id);
+    let line = parseInt((id / 4) + 1);
+
+    switch(direction) {
+        case "ArrowDown":
+            id += 4;
+            if(id > 15) id -= 16;
+
+            break;
+        case "ArrowUp":
+            id -= 4;
+            if(id < 0) id += 16;
+
+            break;
+        case "ArrowLeft":
+            switch(line) {
+                case 1:
+                    id--;
+                    if(id < 0) id += 4;
+
+                    break;
+                case 2:
+                    id--;
+                    if(id < 4) id += 4;
+
+                    break;
+                case 3:
+                    id--;
+                    if(id < 8) id += 4;
+
+                    break;
+                case 4:
+                    id--;
+                    if(id < 12) id += 4;
+
+                    break;
+            }
+
+            break;
+        case "ArrowRight":
+            switch(line) {
+                case 1:
+                    id++;
+                    if(id > 3) id -= 4;
+
+                    break;
+                case 2:
+                    id++;
+                    if(id > 7) id -= 4;
+
+                    break;
+                case 3:
+                    id++;
+                    if(id > 11) id -= 4;
+
+                    break;
+                case 4:
+                    id++;
+                    if(id > 15) id -= 4;
+
+                    break;
+            }
+
+            break;
+        case "Enter":
+            if(!isFound(id)) flipCard(selected_card);
+            else alert("Already flipped!");
+        break;
+    }
+
+    //while(isFound(id)) {
+    //    id = findBestCard(id, direction);
+    //}
+
+    selected_card = img_array[id];
+
+    if(!isFound(id)) setCardBrightness(115);
+    else setCardBrightness(85);
+    
+    event.preventDefault();
```

Comment:
@vzazueta can we change the `preventDefault` to only the navigation arrows? Otherwise it will prevent the default of all keydown keys.

Reply:
Will look into it!